src/fourier.jl

```diff
@@ -0,0 +1,781 @@
+function positive_fft_bins(n_bin::Integer; include_zero::Bool = false)
+    minbin = 2
+    if include_zero
+        minbin = 1
+    end
+    return (minbin : (n_bin+1) ÷ 2)
+end
+
+function poisson_level(norm::String; meanrate = nothing, n_ph = nothing, backrate::Real = 0.0)
+    if norm == "abs"
+        return 2.0 * meanrate
+    elseif norm == "frac"
+        return 2.0 / (meanrate - backrate)^2 * meanrate
+    elseif norm == "leahy"
+        return 2.0
+    elseif norm == "none"
+        return float(n_ph)
+    else 
+        throw(ArgumentError("Incorrect Arguments"))
```

Comment:
Please use a more descriptive error message.  Since this is an `if` on the value of `norm`, the only incorrect argument is `norm`, right?  It'd also be useful to print it out explicitly, and test also the error paths.